conans/client/generators/__init__.py

```diff
@@ -149,6 +149,7 @@ def write_generators(self, conanfile, old_gen_folder, new_gen_folder, output):
                         generator.generate()
                     continue
                 except Exception as e:
+                    print(traceback.format_exc())
```

Comment:
print

Reply:
I agree this traceback is very useful when something goes wrong with a generator, I am happy to leave it (if we capture ConanException for not showing the trace)